sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs

```diff
@@ -201,8 +201,9 @@ public BlobSasBuilder()
         /// class to create a Blob Service Sas.
         /// </summary>
         /// <param name="permissions">
-        /// The time at which the shared access signature becomes invalid.
-        /// This field must be omitted if it has been specified in an
+        /// The permissions associated with the shared access signature. 
```

Comment:
We need to remove the trailing whitespace so CI can pass.

Reply:
Removed the trailing whitespaces. Please review [this](https://github.com/Azure/azure-sdk-for-net/pull/26570/commits/66865fbe352a467f554f60016f5769c8e9973906) commit once you get a chance.